### ISSUE TITLE
Using parent tags in description as optional conditions for applicability of templates (i.e. dynamic subsets)

### ISSUE THREAD

**Issue by jdshkolnik** [figueiredorui/1-click-child-links#29]
Hi @figueiredorui,

I've been thinking about how to signify I only want a subset of templates for a particular parent work item type. I feel the use of tags might resolve this and be in line with how you did parent work item applicability, like [Bug] in the template description.

I propose supporting optional tags in the template description, like {tag1, tag2} which would mean "only if tag1 and tag2" exist on parent. Thoughts?

**Comment by jdshkolnik** [comment]
@figueiredorui Would you be open to someone contributing a pull request that extends the applicability values in the template description? Perhaps something like this?

``` json
{
  "applywhen": [
    {
      "System.WorkItemType": "Product Backlog Item",
      "System.State": "Approved",
      "System.Tags" : ["Blah", "ClickMe"]
    },
    {
      "System.WorkItemType": "Custom Type",
      "System.State": "Custom State",
      "System.Title": "Repeatable item"
    }
  ]
}
```

It'd obviously be minified to a single line.

**Comment by figueiredorui** [comment]
Released v10.0
